### DISCUSSION TITLE
Welcome to FFMPEG-Audio-Encoder Discussions!

### DISCUSSION THREAD

**Discussion by jessielw** [jessielw/FFMPEG-Audio-Encoder#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ossnorry** [comment]
But of a small feature request, would love to see vorbis support. Plenty of tools require vorbis, or in some cases it's preferable to plus in constrained environments, and possibly some of the additional encoding options for both vorbis and opus. The possibility of being able to export them into an off container would be nice.

Semi-related, do you have a Patreon or otherwise easy way to support you? A couple people I work with on small projects struggle with command line tools, and if being able to support you in ways that could enable new features is something I'm up for. 

Thanks for the great project! ❤️

**Reply by jessielw** [reply]
Hello, I actually initially wanted to add Opus/Vorbis official tools instead of FFMPEG's built in one. However, with work and other projects it's been low on my to do list unfortunately. 

Would type of features would you like to see from these in specific? 

What exactly do you mean by an off container? 

I do not have Patreon, but I do accept Stripe for payments/incentives to prioritize some work. This would be great if we can further discuss what you are looking for etc.
